### PR TITLE
End of line formatting

### PR DESCRIPTION
**Pull request by cyenyxe**
The end of line formatting is inconsistent across the specification document. Some paragraphs are split in multiple lines, whereas others are written in a single line.

This makes a bit too difficult to review some changes where not only the content is modified, but also the line splits. This PR makes converts all the paragraphs into a single line, so any future changes affect only the content.

### CODE REVIEW

**Comment by jkbonfield** [comment]
For what it's worth, my personal preference is one line per sentence rather than per paragraph, but I'm not a VCF maintainer so don't mind either way.

File: VCFv4.3.tex [hunk 1]
```diff
@@ -72,20 +65,16 @@ \subsection{An example}
...
-character encoding that is a strict superset of 7-bit ASCII and has the 
-property that none of the bytes in any multi-byte characters are 7-bit ASCII 
-bytes. As a result, most software that processes VCF files does not have 
-to be aware of the possible presence of multi-byte UTF-8 characters. 
-Note that non-printable characters U+0000-U+0008, U+000B-U+000C, U+000E-U+001F are disallowed.
-Line separators must be CR+LF or LF and they are allowed only as line separators at
-end of line.
-Characters with special meaning (such as field delimiters ';' in INFO 
-or ':' FORMAT fields) must be represented using the capitalized percent encoding:
+The character encoding of VCF files is UTF-8. UTF-8 is a multi-byte character encoding that is a strict superset of 7-bit ASCII and has the property that none of the bytes in any multi-byte characters are 7-bit ASCII bytes.
```
Comment (lbergelson) [inline VCFv4.3.tex]: Missing linebreak?

File: VCFv4.3.tex [hunk 2]
```diff
@@ -72,20 +65,16 @@ \subsection{An example}
...
+As a result, most software that processes VCF files does not have to be aware of the possible presence of multi-byte UTF-8 characters.
+Note that non-printable characters U+0000-U+0008, U+000B-U+000C, U+000E-U+001F are disallowed. Line separators must be CR+LF or LF and they are allowed only as line separators at end of line.
```
Comment (lbergelson) [inline VCFv4.3.tex]: Same here

File: VCFv4.3.tex [hunk 3]
```diff
@@ -102,37 +91,26 @@ \subsection{Character encoding, non-printable characters and characters with spe
...
-Data types supported by VCF are: Integer (32-bit, signed), Float (32-bit, formatted 
-to match the regular expression \texttt{\^{}[-+]?[0-9]*\textbackslash.?[0-9]+([eE][-+]?[0-9]+)?\$}, \texttt{NaN}, or \texttt{+/-Inf}), Flag, Character, and
-String. For the Integer type, the values from $-2^{31}$ to $-2^{31}+7$ cannot be stored in the binary version and therefore 
-are disallowed in both VCF and BCF, see \ref{BcfTypeEncoding}.
+Data types supported by VCF are: Integer (32-bit, signed), Float (32-bit, formatted to match the regular expression \texttt{\^{}[-+]?[0-9]*\textbackslash.?[0-9]+([eE][-+]?[0-9]+)?\$}, \texttt{NaN}, or \texttt{+/-Inf}), Flag, Character, and String.
+For the Integer type, the values from $-2^{31}$ to $-2^{31}+7$ cannot be stored in the binary version and therefore are disallowed in both VCF and BCF, see \ref{BcfTypeEncoding}.
 
 \subsection{Meta-information lines}
+File meta-information is included after the \#\# string and must be key=value pairs.
+Meta-information lines are optional, but if they are present then they must be completely well-formed. Note that BCF, the binary counterpart of VCF, requires that all entries are present.
```
Comment (lbergelson) [inline VCFv4.3.tex]: another

File: VCFv4.3.tex [hunk 4]
```diff
@@ -102,37 +91,26 @@ \subsection{Character encoding, non-printable characters and characters with spe
...
-\#\#FILTER, etc.), extra fields can be included after the default fields. For example:
+All structured lines that have their value enclosed within "$<>$" require an ID which must be unique within their type.
+For all of the structured lines (\#\#INFO, \#\#FORMAT, \#\#FILTER, etc.), extra fields can be included after the default fields.
+For example:
 \begin{verbatim}
 ##INFO=<ID=ID,Number=number,Type=type,Description="description",Source="description",Version="128">
 \end{verbatim}
-In the above example, the extra fields of ``Source'' and ``Version'' are
-provided. Optional fields must be stored as strings even for numeric values.
+In the above example, the extra fields of ``Source'' and ``Version'' are provided. Optional fields must be stored as strings even for numeric values.
```
Comment (lbergelson) [inline VCFv4.3.tex]: breka

File: VCFv4.3.tex [hunk 5]
```diff
@@ -364,15 +352,25 @@ \subsubsection{Fixed fields}
   \end{table}
 
   The exact format of each INFO sub-field should be specified in the meta-information (as described above).
-  Example for an INFO field: DP=154;MQ=52;H2. Keys without corresponding values may be used to indicate group membership (e.g. H2 indicates the SNP is found in HapMap 2). See Section~\ref{sv-info-keys} for additional reserved INFO sub-fields used to encode structural variants.
+  Example for an INFO field: DP=154;MQ=52;H2. Keys without corresponding values may be used to indicate group membership (e.g. H2 indicates the SNP is found in HapMap 2).
```
Comment (lbergelson) [inline VCFv4.3.tex]: break

File: VCFv4.3.tex [hunk 6]
```diff
@@ -580,16 +596,11 @@ \section{FORMAT keys used for structural variants}
...
-alternate alleles or because the haplotypes cannot be determined). CN specifies
-the integer copy number of the variant in this sample. CNQ is encoded as a
-phred quality $-10log_{10}$ p(copy number genotype call is wrong). CNL
-specifies a list of $log_{10}$ likelihoods for each potential copy number,
-starting from zero. CNP is 0 to 1-scaled copy number posterior
-probabilities (and otherwise defined precisely as the CNL field), intended to
-store imputed genotype probabilities. When possible, GT/GQ/GL/GP should be used
-instead of (or in addition to) these keys.
+These keys are analogous to GT/GQ/GL/GP and are provided for genotyping imprecise events by copy number (either because there is an unknown number of alternate alleles or because the haplotypes cannot be determined).
+CN specifies the integer copy number of the variant in this sample. CNQ is encoded as a phred quality $-10log_{10}$ p(copy number genotype call is wrong).
```
Comment (lbergelson) [inline VCFv4.3.tex]: break

File: VCFv4.3.tex [hunk 7]
```diff
@@ -1465,15 +1536,17 @@ \subsubsection{Genotype encoding}
...
+The encoding is as follows:
 
 \vspace{0.3cm}
 \textbf{BCF2 type descriptor byte}
 
 \vspace{0.3cm}
 \begin{tabular}{|p{2cm} | p{10cm}|} \hline
 Bit & Meaning \\ \hline
-5,6,7,8 bits & The number of elements of the upcoming type.  For atomic values, the size must be 1.  If the size is set to 15, this indicates that the vector has 15 or more elements, and that the subsequent BCF2 byte stream contains a typed Integer indicating the true size of the vector.  If the size is between 2-14, then this Integer is omitted from the stream and the upcoming stream begins immediately with the first value of the vector.  A size of 0 indicates that the value is MISSING. \\ \hline
+5,6,7,8 bits & The number of elements of the upcoming type. For atomic values, the size must be 1. If the size is set to 15, this indicates that the vector has 15 or more elements, and that the subsequent BCF2 byte stream contains a typed Integer indicating the true size of the vector. If the size is between 2-14, then this Integer is omitted from the stream and the upcoming stream begins immediately with the first value of the vector.  A size of 0 indicates that the value is MISSING. \\ \hline
```
Comment (lbergelson) [inline VCFv4.3.tex]: some breaks in here

**Comment by cyenyxe** [comment]
@lbergelson I have made all the suggested changes except the last one; adding linebreaks to the table decreases readability a lot and makes the table formatting quite confusing.

If you are okay with that please go ahead and merge the PR.

**Comment by jmarshall** [comment]
This rebase brought back the "relation ship" typo.

**Comment by cyenyxe** [comment]
I have cherry-picked the commit because I don't know how to partially undo the rebase :(

**Comment by lbergelson** [comment]
@cyenyxe I rebased your branch so that it's a single commit and doesn't need the cherry pick.  I put it here `lb_rebase_formatting_branch`.  If you want you can reset your branch to that, or we could merge that as a different pr.

**Comment by cyenyxe** [comment]
Now using single commit by @lbergelson after verifying that there were no differences in the .tex file.

**Pull request merged**